README.md

```diff
@@ -7,74 +7,4 @@ in this directory, or on your course webpage.
 The remainder of this README is the user-facing documentation for the analysis
 you will build.
 
-You should remove, from your version, everything above and including this line.
```

Comment:
I'm surprised that you deleted this and everything _below_ it, rather than _above_. Was there a particular reason?